src/DocBlock/Tags/See.php

```diff
@@ -56,6 +56,10 @@ public static function create(
         $parts       = preg_split('/\s+/Su', $body, 2);
         $description = isset($parts[1]) ? $descriptionFactory->create($parts[1], $context) : null;
 
+        if (preg_match('{^https?://}', $parts[0])) {
+            return null;
```

Comment:
Another kinda related issue is `@var array<int, int>` also fails with `Invalid Fqsen array<int`, despite it being a valid type declaration.. I think Fqsen is just not flexible enough or something.